src/app/src/components/GroupedBarChart.js

```diff
@@ -52,7 +54,9 @@ const GroupedBarChart = ({ items }) => {
                     tickSize: 5,
                     tickPadding: 5,
                     tickRotation: 0,
+                    format: formatValue,
                 }}
+                tooltipFormat={formatValue}
```

Comment:
For "number types", (e.g., "What is your gender"), this formatValue doesn't seem to be getting applied on the tooltip. It looks right on the axis, though. Looking at the `formatValue` output, it seems like it is set up correctly. This may be a bug deeper in the library? Seems to be working fine for the pct type.

![image](https://user-images.githubusercontent.com/1014341/103916743-c7327f00-50da-11eb-88c1-3cc0ed25ea29.png)

Reply:
Fixed in ca1cb5e. It looks like the label is getting passed the values as a string instead of as a number, so I'm parsing the values now to return them to numbers for proper formatting.